src/common/FastRational.h

```diff
@@ -626,6 +676,8 @@ inline void addition(FastRational& dst, const FastRational& a, const FastRationa
             dst.den = zd;
         }
         dst.kill_mpq();
+        dst.setWordPartValid();
+////        assert(dst.isWellFormed());
```

Comment:
I agree that `dst.setWordPartValid` should be called (it seems there was an implicit assumption that this would be always true).
Why is the `assert` commented out? I would keep it.

Reply:
Now I am confused, `kill_mpq` should set the flag `wordPartValid` if this was not the case, no?

Reply:
I believe that you're right.  I'll try to do that.